src/main/java/org/littleshoot/proxy/impl/ClientToProxyConnection.java

```diff
@@ -22,18 +22,17 @@
 import io.netty.handler.traffic.GlobalTrafficShapingHandler;
 import io.netty.util.concurrent.Future;
 import io.netty.util.concurrent.GenericFutureListener;
-
```

Comment:
@jekh We should use common formatting rules, which are integrated with the build so it not depends the IDE.